pure_pursuit_control:
  ros__parameters:
    look_ahead: 0.35
    wheelbase: 0.325
    curvature_gain: 0.5
    steering_limit: 0.35
    fast_speed: 1.0
    slow_speed: 0.4
    dynamic_path_timeout: 0.25
    use_path_topic: true
    selected_path_topic: "selected_path"
    static_waypoints_file: "xyhead_demo_pp.csv"
    log_debug: false
